cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -31,27 +31,26 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
     }
     section("Listing databases", "administration-databases-show-databases") {
```

Comment:
From the trello-card: `No descriptions for columns in SHOW databases? (pending HF work?)` (I could not see that it was fixed)